### PR TITLE
Various Documentation Fixes

### PR DESCRIPTION
**Pull request by Akbadain29**
### Purpose
Various documentation and formatting fixes.

### CODE REVIEW

File: docs/development/How-To-User-Interface.md [hunk 1]
```diff
@@ -16,11 +18,11 @@ Make sure to attach the relevant GUI Script component to the prefab itself (In t
 
 ### GUI Script Tips
 
-To be expanded. Include Netpage assignment variable declaration, ultisation of UpdateDisplay(), declartion in NetTab.cs for NetTabType enumerical  and communication to backend script.
+To be expanded. Include Netpage assignment variable declaration, ultilisation of UpdateDisplay(), declaration in NetTab.cs for NetTabType enumerical  and communication to backend script.
```
Comment (Bod9001) [inline docs/development/How-To-User-Interface.md]: is it supposed to be utilisation?

Comment (Akbadain29) [reply docs/development/How-To-User-Interface.md]: Yes, sorry.

Comment (Akbadain29) [reply docs/development/How-To-User-Interface.md]: I don’t know how to approve the changes on my phone, but your correction is right.

Comment (corp-0) [inline docs/development/How-To-User-Interface.md]:
```suggestion
To be expanded. Include Netpage assignment variable declaration, utilisation of UpdateDisplay(), declaration in NetTab.cs for NetTabType enumerical  and communication to backend script.
```

**Pull request merged**